reports/management/commands/import-historical.py

```diff
@@ -11,17 +11,34 @@ class Command(BaseCommand):
     def add_arguments(self, parser):
         parser.add_argument('filenames', nargs='+', help='Excel file(s) to import')
         parser.add_argument('--global', action='store_true', help='Coverage is global')
-        parser.add_argument('--region REGION', action='store', dest='region', help='Import historical data for a region. One of: %s' % ', '.join(sorted(REGION_COUNTRY_MAP.keys())))
-        parser.add_argument('--country COUNTRY', action='store', dest='country', help='Import historical data for a country. One of: %s' % ', '.join(sorted(c[0] for c in get_countries())))
+        parser.add_argument(
+            '--region REGION',
+            action='store',
+            dest='region',
+            help='Import historical data for a region. One of: %s' % ', '.join(sorted(REGION_COUNTRY_MAP.keys())))
+        parser.add_argument(
+            '--country COUNTRY',
+            action='store',
+            dest='country',
+            help='Import historical data for a country. One of: %s' % ', '.join(sorted(c[0] for c in get_countries())))
+        parser.add_argument(
+            '--year YEAR',
+            action='store',
+            dest='year',
+            help='State the year the imported file belongs to. Either "2010" or "2015"')
 
 
     def handle(self, *args, **options):
         historical = Historical()
         coverage = None
         region = None
         country = None
+        year = options['year']
         filenames = options['filenames']
 
+        if not year in ['2010', '2015']:
+            raise CommandError(f"Invalid year: {year}")
```

Comment:
Default to current historical file i.e. 2015

Reply:
QQ @kilemensi: instead of raising `CommandError` we should default to the current year. What if the `sheet` supplied does not conform to the current historical?

Reply:
I'm assuming it will fail @ascii-dev ... If you're running this command in 2020 chances are you want to import 2015 only in rare cases you'd want to import 2010 data so that is a special case.

In 2025, the default would be 2020, etc.